src/mono-runtimes/mono-runtimes.targets

```diff
@@ -301,18 +299,24 @@
         Directories="$(MonoSourceFullPath)\sdks\out"
     />
     <DownloadUri
-        SourceUris="https://xamjenkinsartifact.blob.core.windows.net/mono-sdks/$(_MonoArchiveName).tar.gz"
-        DestinationFiles="$(AndroidToolchainCacheDirectory)\$(_MonoArchiveName).tar.gz"
+        SourceUris="https://xamjenkinsartifact.blob.core.windows.net/mono-sdks/$(_MonoArchiveName).zip"
```

Comment:
this should be using the CDN endpoint `xamjenkinsartifact.azureedge.net` instead.